src/H5Fquery.c

```diff
@@ -143,8 +143,8 @@ H5F_get_actual_name(const H5F_t *f)
  * Function:	H5F_get_extpath
  *
  * Purpose:	Retrieve the file's 'extpath' flags
- *		This is used by H5L_extern_traverse() and H5D_build_extfile_prefix() to retrieve the main file's
- *location when searching the target file.
+ *		This is used by H5L_extern_traverse() and H5D_build_extfile_prefix() to retrieve the main
+ *file's location when searching the target file.
```

Comment:
Should have space between * and file's?

Reply:
Remember comments are not changed except for line lenght.

Reply:
Also that spacing matches 1.12 and dev.

Reply:
The actual cause is there are tabs in the comment, which messes with the line length!
I will need to create an issue for fixing comments.